src/Kerbero.WebApi/Controllers/GetNukiSmartLockController.cs

```diff
@@ -0,0 +1,33 @@
+using Kerbero.Domain.Common.Interfaces;
+using Kerbero.Domain.NukiActions.Interactors;
+using Kerbero.Domain.NukiActions.Models;
+using Kerbero.WebApi.Utils.Extensions;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Kerbero.WebApi.Controllers;
+
+[ApiController]
+[Route("nuki/smartlock")]
+public class GetNukiSmartLockController: ControllerBase
+{
+	private readonly InteractorAsync<NukiAuthenticatedRequestDto, NukiSmartLocksListPresentationDto> _nukiSmartLocksListInteractor;
+
+	public GetNukiSmartLockController(InteractorAsync<NukiAuthenticatedRequestDto,
+		NukiSmartLocksListPresentationDto> nukiSmartLocksListInteractor)
+	{
+		_nukiSmartLocksListInteractor = nukiSmartLocksListInteractor;
+	}
+
+	[HttpGet]
+	public async Task<ActionResult> GetSmartLocksListByKerberoAccount(int kerberoAccountId)
+	{
+		var interactorResponse = await _nukiSmartLocksListInteractor.Handle(new NukiAuthenticatedRequestDto(kerberoAccountId));
+		if (interactorResponse.IsSuccess)
+		{
+			return Ok(interactorResponse.Value);
+		}
+
+		var error = interactorResponse.Errors.First();
```

Comment:
```suggestion
		var error = interactorResponse.Errors.First();
```
siamo sicuri ci sia sempre almeno un errore dentro Errors?

Reply:
Sì e se si spacca, meglio! Così mi accorgo che c'è una casistica non gestita.